webapp/src/components/NewsList.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <news-item
+        v-for="newsItem in sortedItems"
+        v-bind:title="newsItem.title"
+        v-bind:votes="newsItem.votes"
+        v-bind:key="newsItem.title"
+        v-on:upvoteItem="upvoteNews"
+        v-on:downvoteItem="downvoteNews"
+        v-on:removeNews="removeNews"
+    ></news-item>
+    <div class="form-div">
```

Comment:
You could have a `NewsForm` component. In your case, you don't want to allow the same `title`. This component would be responsible to show validation errors along with the form. It receives the `uniqueTitles` as a data-binding and shows a validation error when you try to add a title that already exist. It's good user experience to show an explanation *why* something is not possible. Don't frustrate the end user.